mintapi/signIn.py

```diff
@@ -510,7 +510,12 @@ def mfa_page(
     if mfa_method is None:
         mfa_result = search_mfa_method(driver)
     else:
-        mfa_result = set_mfa_method(driver, mfa_method)
+        try:
+            mfa_result = set_mfa_method(driver, mfa_method)
+        except Exception as e:
```

Comment:
Is there a way we can actually detect the success, or catch a more specific exception, vs just catching all exceptions?

Reply:
The `set_mfa_method` catches the actual exception and re-raising a `RuntimeException`. So I feel we have three options:
1. Create a specific exception type for this that should be thrown from the `set_mfa_method` and caught here. I personally prefer this option.
2. Instead of re-raise a `RuntimeException` in the `set_mfa_method` we could just re-raise the actual exception so we can keep the actual type. The problem with this would be that in this part of the code that type of the exception would make not that much sense (`NoSuchElementException`).
3. I can catch `RuntimeException` here

Any thoughts on which one would you prefer or any other ideas for other options?

Reply:
I vote option 1.